src/Core.Nhibernate/Core.Nhibernate.csproj

```
@@ -1,142 +1,27 @@
-    <Compile Include="Extensions\StringExtensions.cs" />
-    <Compile Include="NhibernateConfig\MappingHelper.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\ClientCorsOriginMappingOverride.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\ClientCustomGrantTypeMappingOverride.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\ClientIdPRestrictionMappingOverride.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\ClientMappingOverride.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\ClientPostLogoutRedirectUriMappingOverride.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\ClientRedirectUriMappingOverride.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\ClientScopeMappingOverride.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\ClientSecretMappingOverride.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\ConsentMappingOverride.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\ScopeClaimMappingOverride.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\ScopeMappingOverride.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\ScopeSecretMappingOverride.cs" />
-    <Compile Include="NhibernateConfig\MappingOverrides\TokenMappingOverride.cs" />
-    <Compile Include="Serialization\AsyncHelper.cs" />
-    <Compile Include="Serialization\ClaimConverter.cs" />
-    <Compile Include="Serialization\ClaimsPrincipalConverter.cs" />
-    <Compile Include="Serialization\ClientConverter.cs" />
-    <Compile Include="Serialization\ScopeConverter.cs" />
-    <Compile Include="Services\ClientConfigurationCorsPolicyService.cs" />
-    <Compile Include="Stores\ClientStore.cs" />
-    <Compile Include="NhibernateServiceOptions.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Stores\AuthorizationCodeStore.cs" />
-    <Compile Include="Stores\BaseTokenStore.cs" />
-    <Compile Include="Stores\ConsentStore.cs" />
-    <Compile Include="Stores\NhibernateStore.cs" />
-    <Compile Include="Stores\RefreshTokenStore.cs" />
-    <Compile Include="Stores\ScopeStore.cs" />
-    <Compile Include="Stores\TokenHandleStore.cs" />
-    <Compile Include="TokenCleanup.cs" />
   </ItemGroup>
-  <ItemGroup />
   <ItemGroup>
     <None Include="..\default.licenseheader">
       <Link>default.licenseheader</Link>
     </None>
-    <None Include="packages.config" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <ItemGroup>
+    <PackageReference Include="Autofac" Version="6.4.0" />
+    <PackageReference Include="AutoMapper" Version="10.0.0.0" />
+    <PackageReference Include="FluentNHibernate" Version="2.1.2" />
+    <PackageReference Include="IdentityServer3" Version="2.6.3" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Please drop package versions to the version used in FW 5.1. 
I'd rather not have this package dictating the versions we're using.
```suggestion
    <PackageReference Include="Autofac" Version="6.2.0" />
    <PackageReference Include="AutoMapper" Version="10.1.1" />
    <PackageReference Include="FluentNHibernate" Version="2.1.2" />
    <PackageReference Include="IdentityServer3" Version="2.6.3" />
    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.1" />
    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

---

I'd like to update Autofac to 6.4.0 and Newtonsoft.Json to 13.0.1 (Newtonsoft particularly due to know issue prior to 13.0.1). AutoMapper version here isn't relevant as it's not included as a dependency for the library, it's merged into the DLL.